Atomic/util.py

```diff
@@ -331,13 +331,22 @@ def skopeo_subprocess(cmd):
         raise ValueError(results.stderr)
     return results
 
+def skopeo_call(cmd):
+    try:
+        return_code = call(cmd, stdout=sys.stdout, stderr=sys.stderr, stdin=sys.stdin)
+    except Exception as e: # pylint: disable=broad-except
+        raise Exception(e)
+    return return_code
+
 def skopeo_standalone_sign(image, manifest_file_name, fingerprint, signature_path, debug=False):
     cmd = ['skopeo']
     if debug:
         cmd = cmd + ['--debug']
     cmd = cmd + ['standalone-sign', manifest_file_name, image,
                  fingerprint, "-o", signature_path]
-    return skopeo_subprocess(cmd)
+    return_code = call(cmd)
```

Comment:
I was accounting for the case where an exception may not be raised but the command actually failed.  Not worth it?